desktop/assets/texts/ar-SA/preferences_dialog.ftl

```diff
@@ -20,6 +20,13 @@ storage-backend-memory = الذاكرة
 recent-limit = حد الملفات الحديثة
 recent-clear = إمسح
 theme = المظهر
-theme-system = الإعدادات الإفتراضية
+theme-system = الإعدادات المبدئية
 theme-light = فاتح
 theme-dark = داكن
+# See for context https://github.com/FeralInteractive/gamemode
+gamemode = وضع اللعبة
```

Comment:
I'm not sure about this, it seems fully translated instead of transliterated. Added a comment on crowdin.

Reply:
Undid it until is cleared up.

Reply:
You'd also have to undo the translations below that have GameMode in them

Reply:
Fair, done.